app/utils/images.js

```diff
@@ -1,11 +1,21 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {
+    IMAGE_MAX_HEIGHT,
+    IMAGE_MIN_DIMENSION,
+} from 'app/constants/image';
+
 let previewComponents;
-const IMAGE_MAX_HEIGHT = 350;
-const IMAGE_MIN_DIMENSION = 50;
 
 export const calculateDimensions = (height, width, viewPortWidth = 0, viewPortHeight = 0) => {
+    if (height === 0 || width === 0) {
+        return {
+            height: IMAGE_MIN_DIMENSION,
+            width: IMAGE_MIN_DIMENSION,
```

Comment:
Why return a default dimension that leads to showing empty image view for invalid image URL instead of returning null so that those calling this function could conditionally render an image or not?